src/LedgerLivePlatformSDK/storage.ts

```diff
@@ -0,0 +1,47 @@
+/**
+ * Storage interface to access Wallet Server store.
+ */
+export interface StorageSDK {
+  /**
+   * Store the value with the associated key.
+   * @param key - Key to find later the value
+   * @param value - Value to save
+   */
+  save(key: string, value: string): Promise<void>;
```

Comment:
Since there is no return value, I guess the idea is that the method throws an error in case the save was not possible (for any reason) on the wallet side?

Reply:
Yes. I update the doc.